odata_api/src/main/java/com/sdl/odata/api/processor/ProcessorResult.java

```diff
@@ -33,7 +34,7 @@
 public final class ProcessorResult {
 
     private final ODataResponse.Status status;
-    private final Object data;
+    private final QueryResult data;
```

Comment:
Will be good to update the variable name here and also fix the java docs in methods accordingly.